Header header # this field contains time stamp 
string camera_name
int32 camera_id
sensor_msgs/Image rgb_data
sensor_msgs/Image depth_map
sensor_msgs/Image segmentation_labels
sensor_msgs/Image height_map
int64[] biomass_estimate
float64 latitude
float64 longitude
float64 gps_heading